resume-buddy/src/main/java/com/google/sps/data/EnumUtility.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.data;
+
+public class EnumUtility {
```

Comment:
So enums are classes themselves and don't need to live in this `EnumUtility` class.

I actually think the best thing to do is to put each enum into its own file in `data/`. For example, `Degree.java` will look like this:

```
package com.google.sps.data;

public enum Degree {
  HIGH_SCHOOL,
  ASSOCIATE,
  BACHELOR,
  MASTER,
  DOCTORATE,
  NO_PREFERENCE;
}
```

Then you don't have to use the `EnumUtility.-` prefix when using any of these enums.